docker-compose/docker-compose.yml

```diff
@@ -38,7 +38,7 @@ services:
   # WHISK CONTROLLER
   controller:
     image: openwhisk/controller
-    command: /bin/sh -c "controller/bin/controller 0 >> /logs/controller-local_logs.log 2>&1"
+    command: /bin/sh -c "./init.sh 0 >> /logs/controller-local_logs.log 2>&1"
```

Comment:
IIRC `controller/bin/controller` should be the `init.sh` script actually

Reply:
controller is the script generated by Gradle plugin. [init.sh][1] invokes this internally

[1]: https://github.com/apache/incubator-openwhisk/blob/master/core/controller/init.sh

Reply:
I've seen a similar problem, so agreeing with @chetanmeh on this.
Maybe the invoker command should use `init.sh` as well.